app/models/user.rb

```diff
@@ -86,13 +86,6 @@ def private_namespace_and_team_available
     if ns.nil?
       errors.add(:username, "'#{username}' cannot be transformed into a " \
         "valid namespace name")
-    elsif Namespace.exists?(name: ns)
-      clar = (ns != username) ? " (modified so it's valid)" : ""
-      errors.add(:username, "cannot be used: there is already a namespace " \
-        "named '#{ns}'#{clar}")
-    elsif Team.exists?(name: username)
-      errors.add(:username, "cannot be used: there is already a team named " \
-        "like this")
```

Comment:
Do these checks really hurt ? If not, then I'd leave them just to be sure.

Reply:
The problem is that it will never happen now that there's no name clashes. I know this method is called before `create_personal_namespace!` but it doesn't make sense to show an error about name conflicts if we are trying to solve the conflicts.

Reply:
Gotcha